io.github.rafaelfassi.QLogExplorer.yaml

```diff
@@ -0,0 +1,33 @@
+app-id: io.github.rafaelfassi.QLogExplorer
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+rename-appdata-file: qlogexplorer.appdata.xml
+rename-desktop-file: qlogexplorer.desktop
+rename-icon: qlogexplorer
+command: qlogexplorer
+finish-args:
+  # Required for Qt render.
+  - --socket=wayland
+  - --device=dri
+  # Necessary if X11 fallback takes place.
+  - --socket=fallback-x11
+  - --share=ipc
+  # Necessary to open system log files.
+  - --filesystem=host
+  # Local sockets are used to detect other running instance and forward any open file request to it.
+  # Qt requires Network module to use sockets. Even though an error message is print on app startup
+  # when no network permission is granted, it's working properly.
+  # - --share=network
+modules:
+  - name: qlogexplorer
+    buildsystem: simple
```

Comment:
you should use `buildsystem: cmake-ninja` and then add extra stuff to the `post-install`

Reply:
Thanks for pointing that. Done!